scripts/build-es-shims-data/index.js

```diff
@@ -5,6 +5,19 @@ const path = require("path");
 const { generateData, environments, writeFile } = require("./utils-build-data");
 
 const newData = generateData(environments, require(`./es-shims-features`));
+
+// These features are missing from compat-table. Remove this from this list once they are added.
+const missing = [
+  "Array.prototype.toReversed",
+  "Array.prototype.toSorted",
+  "Array.prototype.toSpliced",
+  "Array.prototype.with",
+];
```

Comment:
`compat-table` does not have entries for this proposal yet.

Reply:
That’s because they aren’t shipped anywhere, so everything needs them. Perhaps it should just assume that when data is missing?

Reply:
Yes, the currently have `{}` as "support data" so they are always included. I added them to this list because the current build process warns when using a feature not present in compat-table (so that we don't forget about wiring them).